roles/nameserver/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+# These defaults are present to allow certain tasks to no-op if a secrets repo
+# hasn't been defined. If you want to override these, do so in the secrets repo
+# itself. We override these in  $repo/ansible/inventory/group_vars/nameserver.yml
+secrets_repo:
+  name: UNDEFINED
```

Comment:
No specific reason.  I took it from https://github.com/ceph/ceph-cm-ansible/blob/master/roles/cobbler/defaults/main.yml#L6 originally and used it in the gateway role as well but I see where you actually check for "UNDEFINED" in the cobbler role.

It may not be needed at all since the play will bail out if there's no secrets repo anyway.